CHANGES.rst

```diff
@@ -31,6 +31,14 @@
 - Remove some internal compatibility shims that are no longer
   necessary. See `PR 82 <https://github.com/zopefoundation/persistent/pull/82>`_.
 
+- The precision of ``TimeStamp`` objects created from 6 arguments with
```

Comment:
I will clarify that this refers to the return of the `second()` function: the internal precision used for the `raw()` is unchanged.